jf_agent/main.py

```diff
@@ -47,6 +47,9 @@
 from jf_ingest.jf_jira import load_and_push_jira_to_s3
 from jf_ingest.config import IngestionType, IngestionConfig
 
+# Configure structlog before any loggers are created
+agent_logging.configure_structlog()
```

Comment:
Q: Do we want structlogging from any non-`main.py` entrypoints? If so, consider initializing in the agent logging file.

This might be the right way to do things, just food for thought here.